src/NServiceBus.RabbitMQ/Routing/ConventionalRoutingTopology.cs

```diff
@@ -74,42 +74,37 @@ public void RawSendInCaseOfFailure(IModel channel, string address, byte[] body,
 
         public void Initialize(IModel channel, string mainQueue)
         {
-
             CreateExchange(channel, mainQueue);
             channel.QueueBind(mainQueue, mainQueue, string.Empty);
-
         }
 
-        static string ExchangeName(Type type)
-        {
-            return type.Namespace + ":" + type.Name;
-        }
+        static string ExchangeName(Type type) => type.Namespace + ":" + type.Name;
```

Comment:
The problem with string interpolation is that, before .net 4.6, there's no nice way to make it culture invariant. In .net 4.6, we can use [FormattableString.Invariant](https://msdn.microsoft.com/en-us/library/system.formattablestring.invariant(v=vs.110).aspx). Whilst we're on .net 4.5 I think we should avoid string interpolation unless we explicitly want to use the current culture.

A micro-optimisation here would be to use `string.Concat()`, since the current use of two `+` operators forces an extra heap allocation.